src/assets/usage-examples-code/sprites/free_sprite/free_sprite-3-free-sprite.cpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * Usage Example: freeing a sprite
+ **/
+
+#include "splashkit.h"
+
+int main()
+{
+    // widow to draw the sprite on
+    window start = open_window("create_sprite", 600, 600);
+
+    // bitmap for creating a sprite
+    bitmap player = load_bitmap("playerBmp", "protSpriteSheet.png");
```

Comment:
Add the png into the public/usage-examples-images-gifs/sprits

Reply:
I have addde the resource folder